lua/apg/cl_menu.lua

```diff
@@ -229,6 +229,7 @@ local function openMenu( len )
 	saveButton:SetSize( 72, 16 )
 	saveButton:SetText('')
 	saveButton.DoClick = function()
+		if not LocalPlayer():IsSuperAdmin() then return end

```

Comment:
As a server owner, this could be annoying if it's not documented or otherwise made clear that superadmins are exempt from this. If this were made to be a config, or perhaps if it sent a chat message to alert the user that they've been made exempt from this rule, that would be preferable.

Reply:
I'm kind of confused, only super admins can access !apg anyways this (sv_menu.lua ln43), a regular player should not be able to access this in any sort of way already this is just an extra step.

Reply:
Maybe later add a config for players who can access but not really a priority.

Reply:
Fair point. I wasn't taking context into account. Disregard!